### PR TITLE
Update Giphy client; Fixes random button;

### PR DESCRIPTION
**Pull request by rafavalerio**
# To Do
- [x] Update the Giphy client used (`giphy-js-sdk-core` => `@giphy/js-fetch-api`)
- [x] Add a new `I'm Feeling EXTREMELY Lucky` button

# GIF
_This is using the new `EXTREMELY Lucky` button!_ 🤞 

fix

### CODE REVIEW

File: src/components/QuickSearch.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import React from 'react'
-import PropTypes from 'prop-types'
-import styled from 'styled-components'
+import React from "react"
```
Comment (nickspiel) [inline src/components/QuickSearch.js:1]: Can you update your linter to keep the current code style?

File: src/components/QuickSearch.js [hunk 2]
```diff
@@ -14,9 +14,9 @@ const Button = styled.button`
   padding: 0.5rem;
   margin: 0.5rem;
   font-size: inherit;
-  border: solid 1px #74FAA2;
-  box-shadow: 2px 2px 0 0 #74FAA2;
-  color: #74FAA2;
+  border: solid 1px #74faa2;
```
Comment (nickspiel) [inline src/components/QuickSearch.js:17]: Here too.

**Review by nickspiel — APPROVED** [review]
This is awesome! Thanks for the PR. All looks great, would you be able to roll back the linter changes to keep things consistent?

**Comment by rafavalerio** [comment]
So it seems `.prettierrc` was set to double quotes, but the project was mostly using single quotes. I've updated to reflect that and updated everything.

As for the color's case, see this: https://github.com/prettier/prettier/issues/5158
Seems lowercase is the default these days, So I did a full update is that's ok :)

**Comment by nickspiel** [comment]
Absolute legend! Really appreciate the update, the prettier configuration clearly needs a little more work to enforce the style.

**Pull request merged**